core-services/prow/02_config/openshift-hyperfleet/hyperfleet-claude-plugins/_pluginconfig.yaml

```diff
@@ -0,0 +1,39 @@
+approve:
+- commandHelpLink: https://go.k8s.io/bot-commands
+  repos:
+  - openshift-hyperfleet/hyperfleet-claude-plugins
+  require_self_approval: true
+external_plugins:
+  openshift-hyperfleet/hyperfleet-claude-plugins:
+  - endpoint: http://needs-rebase
+    events:
+    - issue_comment
+    - pull_request
+    name: needs-rebase
```

Comment:
Do we need `refresh` to re-trigger the Prow to evaluate the entire PR (including all state checks and hooks) by `/retest` or `/refresh`? 
```
  - endpoint: http://refresh
    events:
    - issue_comment
    name: refresh
```

Maybe we need Re-trigger the Prow to evaluate the entire PR, including all state checks and hooks.